templates/arm-vnet-template.json

```diff
@@ -118,8 +118,21 @@
     "serviceEndpoints": {
       "type": "array",
       "defaultValue": []
+    },
+    "routetableName": {
```

Comment:
shouldn't be a parameter imo better to fix ithc

Reply:
Cheers - now updated